insights/parsers/numeric_user_group_name.py

```diff
@@ -3,26 +3,38 @@
 =========================================================================================
 
 This parser reads the output of ``/usr/bin/grep -c '^[[:digit:]]' /etc/passwd /etc/group``
-and detects whether it is 0 or nonzero.
+and detects whether there are any user or group names that begin with a digit.
 
-The grep command returns the number of matches. It matches for user and group names that
-begin with a number. If it returns 0, then there are no such names, otherwise it returns
-the number of matches (names). The parser only detects whether it is 0 or nonzero so as
-to minimize the amount of collected data.
+The grep command returns the number of matches for each file. It matches for user and
+group names that begin with a number. The parser only detects whether the sum is 0 or
+nonzero so as to minimize the amount of collected data.
 
-The boolean is available in the ``data`` attribute but it should not be accessed directly
-(encapsulation). The parser provides methods for comparisons with bool and casting to bool.
+The boolean answers are available through the getters ``has_numeric_user_group()``,
+``has_numeric_user()``, ``has_numeric_group()`` and ``bad_data()``.
```

Comment:
Where do you suggest to set the line? Throw an exception if at least one file is not available? Or throw an exception only when both files are unavailable?